### PR TITLE
[NG] Final Updates

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR brings some final updates on Meshery page and other issues.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/assets/data/faq/index.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ const data = {
             question: "Does Meshery manage service meshes?",
             category: "Meshery",
             answer: ["Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes."],
-            link: "/projects/meshery",
+            link: "/meshery",
```
Comment (Tanuj22) [inline src/assets/data/faq/index.js:108]: Link should be `/service-mesh-management/meshery`

File: src/sections/Home/Banner-default/index.js [hunk 2]
```diff
@@ -37,10 +37,9 @@ const BannerDefault = () => {
               <Button primary className="banner-btn one" title="Join in" url="/community">
                 <FaMapMarkedAlt size={21} className="icon-left" />
               </Button>
-              <Button primary className="banner-btn two" title="Run Meshery" url="/projects/meshery">
+              <Button primary className="banner-btn two" title="Run Meshery" url="/meshery">
```
Comment (Tanuj22) [inline src/sections/Home/Banner-default/index.js:40]: Same

File: src/assets/data/faq/index.js (hunk 1 above)
Comment (Jashpatel1) [reply src/assets/data/faq/index.js:108]: It has redirects btw

Comment (Tanuj22) [reply src/assets/data/faq/index.js:108]: It is better to keep the real link in the site though

**Comment by Tanuj22** [comment]
@Jashpatel1 can you merge `upstream/layer5-ng` branch here. I want to test something

**Comment by Jashpatel1** [comment]
Is it due to cache? The changes are not reflected.

**Comment by Tanuj22** [comment]
> @Tanuj22 Is it due to cache? The changes are not reflected in preview.

Still figuring it out. Merging for now

**Review by Tanuj22 — APPROVED** [review]

**Comment by Tanuj22** [comment]
Screenshot from 2021-02-05 20-04-07
@Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity

**Comment by Jashpatel1** [comment]
> Screenshot from 2021-02-05 20-04-07
> @Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity

I wonder why is it coming this size. It looked perfect in local.

**Comment by Tanuj22** [comment]
> Screenshot from 2021-02-05 20-04-07
> @Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity

@Jashpatel1 ^^

**Comment by Jashpatel1** [comment]
> > Screenshot from 2021-02-05 20-04-07
> > @Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity
> 
> @Jashpatel1 ^^

I am unable to see this on local.

**Comment by Tanuj22** [comment]
> > > Screenshot from 2021-02-05 20-04-07
> > > @Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity
> > 
> > 
> > @Jashpatel1 ^^
> 
> I am unable to see this on local.

Increase the resolution

**Comment by Jashpatel1** [comment]
> > > > @Jashpatel1 This is looking too weird. According to the design, the SVG should be the background and gray cover with reduced opacity
> > > 
> > > 
> > > @Jashpatel1 ^^
> > 
> > 
> > I am unable to see this on local.
> 
> Increase the resolution
Screenshot from 2021-02-10 22-54-37

**Pull request merged**